old-ui/app/components/copyButton.js

```diff
@@ -24,21 +25,22 @@ CopyButton.prototype.render = function () {
   const copied = state.copied
 
   const message = copied ? 'Copied' : props.title || ' Copy '
+  const defaultCopyStyles = ['clipboard', 'cursor-pointer']
 
   return h('.copy-button', {
     style: {
       display: display || 'flex',
       alignItems: 'center',
     },
   }, [
-
     h(Tooltip, {
       title: message,
     }, [
-      h('i.fa.fa-clipboard.cursor-pointer.color-violet', {
+      h('i', {
         style: {
           marginLeft: '5px',
         },
+        className: props.isWhite ? classNames(defaultCopyStyles.concat(['white'])) : classNames(defaultCopyStyles),
```

Comment:
A more idiomatic (and nicer, IMO) use of `classNames` is to do:

```
classNames(defaultCopyStyles, { white: props.isWhite })
```

Reply:
@fvictorio totally agree, thanks! Fixed